tsconfig.json

```diff
@@ -10,7 +10,7 @@
     "skipLibCheck": true,
     "esModuleInterop": true,
     "allowSyntheticDefaultImports": true,
-    "strict": true,
+    "strict": false,
```

Comment:
Could we change this back to true? I think it would be best for a TypeScript project.

Reply:
good catch! updated to true now :)